src/Button/DigitizeButton/DigitizeButton.jsx

```diff
@@ -21,10 +21,10 @@ import OlInteractionTranslate from 'ol/interaction/translate';
 import OlEventsCondition from 'ol/events/condition';
 
 import ToggleButton from '../ToggleButton/ToggleButton.jsx';
-import MapUtil from '../../Util/MapUtil/MapUtil';
-import StringUtil from '../../Util/StringUtil/StringUtil';
-import AnimateUtil from '../../Util/AnimateUtil/AnimateUtil';
-import Logger from '../../Util/Logger';
+import { MapUtil } from '@terrestris/ol-util';
```

Comment:
Unsure here, but I *think* in terms of bundle size it could be smarter to import the classes via the direct path, e.g. `@terrestris/ol-util/MapUtil/MapUtil`.

Maybe @amir-ba can verify this?

Reply:
I would also suggest we stick to the habit of importing through direct paths. at least until we have `react-geo` in ES6